testsuite/features/support/env.rb

```diff
@@ -26,13 +26,14 @@
 STARTTIME = Time.new.to_i
 Capybara.default_max_wait_time = ENV['CAPYBARA_TIMEOUT'] ? ENV['CAPYBARA_TIMEOUT'].to_i : 10
 DEFAULT_TIMEOUT = ENV['DEFAULT_TIMEOUT'] ? ENV['DEFAULT_TIMEOUT'].to_i : 250
+$is_cloud_provider = ENV["PROVIDER"].include? 'aws'
 
 # QAM and Build Validation pipelines will provide a json file including all custom (MI) repositories
 custom_repos_path = File.dirname(__FILE__) + '/../upload_files/' + 'custom_repositories.json'
 if File.exist?(custom_repos_path)
   custom_repos_file = File.read(custom_repos_path)
   $custom_repositories = JSON.parse(custom_repos_file)
-  $build_validation = true
+  build_validation = $is_cloud_provider
```

Comment:
This is wrong @maximenoel8 , we want `build_validation` to be `true` any time we provide a `custom_repositories.json` file. Not only when we are in cloud.